packages/core/map-types.ts

```diff
@@ -14,4 +14,7 @@ export {
 /**
  * MouseEvent gets emitted when the user triggers mouse events on the map.
  */
-export interface MouseEvent { coords: LatLngLiteral; }
+export interface MouseEvent {
+  coords: LatLngLiteral;
+  placeId: string;
```

Comment:
Because some `MouseEvent` objects do not have a `placeId`, such as when the user clicks on a blank portion of the map, the property on the interface should be optional.
```suggestion
  placeId?: string;
```